nidirect_gp/src/Controller/GpSearchController.php

```diff
@@ -233,6 +261,23 @@ public function handleSearchRequest() {
 
   }
 
+  /**
+   * Update the Google maps api key if necessary.
+   */
+  private function updateApiKey() {
```

Comment:
I'm maybe consider having this outside the GPsearch controller so we can use it across the site.

Reply:
Good thought, have done that now (see new class UpdateConfigFromEnvironment)